lib/search-field/index.tsx

```diff
@@ -13,9 +14,8 @@ import type * as T from '../types';
 const KEY_ESC = 27;
 
 type StateProps = {
-  openedTag: T.Tag | null;
+  openedTag: T.TagName | undefined;
```

Comment:
`null` is a clearer signal than `undefined` so it could be nice to have the selector return that if no tag is open.

fun fact! `undefined` is ironically a value, which in earlier versions of JavaScript could be reassigned. it can still come about in two separate conditions: one from the absence of a value, and two in the passing of the `undefined` value itself.

```ts
const o = {a: undefined};

o.a // undefined
o.b // undefined
o.hasOwnProperty('a') // true
o.hasOwnProperty('b') // false
```

Reply:
Fixed!